### ISSUE TITLE
Legend for line chart

### ISSUE THREAD

**Issue by ymuzikant** [capitalone/react-native-pathjs-charts#141]
Are there any plans to implement a legend for line chart? in case there are multiple lines shown then the legend should show each series/line name

**Comment by HyperSimon** [comment]
+1

**Comment by marzolfb** [comment]
Agreed this would be nice. Would be happy to review PR for anyone who wants to submit one. This one is similar to #105 asking for the same thing but for the pie chart instead of a line chart

**Comment by shrupa** [comment]
Any update regarding this enhancement? I need to show legend in my stock lime graph.
Thanks!
